config/qe-views.yaml

```diff
@@ -1,10 +1,169 @@
 ---
 component_readiness:
-  - name: 4.17-qe-main
+  - name: 4.14-qe-main
     base_release:
+      release: "4.14"
+      relative_start: "2024-10-16T00:00:00Z"
+      relative_end: "2024-11-16T23:59:59Z"
+    sample_release:
+      release: "4.14"
+      relative_start: now-7d
+      relative_end: now
+    variant_options:
+      column_group_by:
+        Architecture: {}
+        Network: {}
+        Platform: {}
+      db_group_by:
+        Architecture: {}
+        FeatureSet: {}
+        Installer: {}
+        Network: {}
+        Platform: {}
+        Suite: {}
+        Topology: {}
+        Upgrade: {}
+      include_variants:
```

Comment:
To show just auto-release you can add this to the views:

```
Procedure:
  - automated-release
```

Reply:
@stbenjam Can you review again, thx